stdlib/asyncio/sslproto.pyi

```diff
@@ -76,7 +76,7 @@ class _SSLProtocolTransport(transports._FlowControlMixin, transports.Transport):
     def get_extra_info(self, name: str, default: Any | None = None) -> dict[str, Any]: ...
     @property
     def _protocol_paused(self) -> bool: ...
-    def write(self, data: bytes | bytearray | memoryview) -> None: ...
+    def write(self, data: bytes | bytearray | memoryview[object]) -> None: ...
```

Comment:
Nit: This would in theory allow something like `memoryview[datetime]`.

```suggestion
    def write(self, data: bytes | bytearray | memoryview[Any]) -> None: ...   # any memoryview format or shape
```

Reply:
I *think* #13683 will take care of this. This is more of a cosmetic issue.

Reply:
This should not result in extra warnings in mypy. User code will still uses `memoryview[int]` etc.